drivers/claims_reporting/app/views/claims_reporting/warehouse_reports/reconciliation/_form.haml

```diff
@@ -0,0 +1,21 @@
+= simple_form_for ClaimsReporting::CpPaymentUpload.new, as: :file, url: url_for(action: :create) do |f|
+  %h3 Upload Reconciliation File
+  = f.input :content, as: :file, label: false
+  = f.submit 'Upload', class: 'btn btn-primary'
+
+:ruby
+  available_months = 36.times.map do |n|
+    date = (Date.current << n).end_of_month
+  end
+  @month ||= available_months.first
```

Comment:
This should be dependent on the unique months of `Health::QualifyingActifity.distinct.pluck(:max_date)`

Reply:
Also, @month, should probably be the most recent month in that set.